src/sec_adapter_soc_provisioning.c

```diff
@@ -153,10 +159,10 @@ bool readNetflixData(Sec_ProcessorHandle* processorHandle, NetflixProvisioning**
 
     if (!esn_container || !encryption_key || !hmac_key || !wrapping_key) {
         SEC_LOG_ERROR("Failed to read Netflix provisioning data");
-        free(esn_container);
-        free(encryption_key);
-        free(hmac_key);
-        free(wrapping_key);
+        SEC_MEMSET_FREE(esn_container, esn_size);
```

Comment:
What are we trying to protect by memsetting all of these provisioning files? They should all be encrypted in host/REE memory and no sensitive data available in the clear.